core/types/transaction.go

```diff
@@ -728,7 +728,7 @@ func (t *L1MessagesByQueueIndex) Pop() {
 	log.Error("Pop() is called on L1MessagesByQueueIndex")
 
 	// this is a logic error, the intention should be "Shift()",
-	// so we will follow the same behavior in Pop
+	// so we will follow the same behavior in Shift
```

Comment:
This is not a typo.

This means: "we will follow the same behavior in Pop (as we do in Shift)".

Reply:
very helpful reminder, thanks.